client/state/login/magic-login/actions.js

```diff
@@ -93,39 +88,36 @@ const authSuccess = () => {
 	};
 };
```

Comment:
Do we really need to have those `authError()` and `authSuccess()` functions ? As far I can see they are only used once in this file.